parsl/providers/kubernetes/kube.py

```diff
@@ -322,7 +321,7 @@ def _create_pod(self,
                                                    claim_name=volume[0])))
 
         metadata = client.V1ObjectMeta(name=pod_name,
-                                       labels={"app": job_name},
+                                       labels={"job_id": job_id},
```

Comment:
might be more informative to get the word parsl in this label name somehow, so that users looking at a pod can see that the job ID comes from parsl rather than from some part of kubernetes - for example, so as to not confuse tech support people when they are told the "job_id" of a pod without the relevant context.

Reply:
Good idea; I'll change the label key to `parsl-job-id`.